module/tty0tty.c

```diff
@@ -342,7 +342,14 @@ static int tty0tty_write(struct tty_struct *tty, const unsigned char *buffer, in
 //        tty->low_latency=1;
 	  if(ttyx != NULL)
 	  {
-		  tty_insert_flip_string(ttyx->port, buffer, count);
+		  if((tty->termios.c_cflag & PARENB) && (tty->termios.c_cflag & PARODD))
```

Comment:
I believe that test `(tty->termios.c_cflag & PARODD)` must be removed to work with ODD and EVEN parities.

Reply:
Ah, my bad, in reality the code I added only works for MARK/SPACE parity, since that's what I've needed to use these past few weeks. A proper even/odd check can be implemented, but it's also possible to just keep my commit + only run it if CMSPAR is set.

Reply:
I believe it doesn't make much sense to use even or odd parity in a null-modem emulator. But using the parity bit as an extra bit of Mark/Space information makes sense. Now I understand your commit, adding the CMSPAR test is enough to do the merge.

Reply:
Agreed. The way I need to make use of serial ports does indeed make use of the parity bit as a ninth bit, and having the ability to toggle it manually is important. I've now changed the PR to only add the flag if CMSPAR (MARK/SPACE) is set.